authorizations/authorization-ranger/src/main/resources/META-INF/services/org.apache.gravitino.connector.authorization.AuthorizationProvider

```diff
@@ -0,0 +1,19 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+org.apache.gravitino.authorization.ranger.RangerAuthorization
```

Comment:
We could only use RangerAuthorization for all catalogs. This seems not ok if we only support partial catalog authorization.

Reply:
We can add multiple Authorization in the `authorization.AuthorizationProvider`, just like 
https://github.com/apache/gravitino/blob/main/core/src/test/resources/META-INF/services/org.apache.gravitino.connector.authorization.AuthorizationProvider#L19

Look more carefully before you jump to conclusions！！！

Reply:
OK, I got it.